docs/setup-and-installing.md

```diff
@@ -2,151 +2,152 @@
+## Set up and get started with the SpatialOS GDK for Unity
 
-1. Clone the repository: `git clone git@github.com:spatialos/UnityGDK.git`  or `git clone  https://github.com/spatialos/UnityGDK.git`
+### Sign up for a SpatialOS account
 
-1. Open the `workers/unity` project in your Unity Editor.
+You'll need a SpatialOS account in order to run the SpatialOS GDK for Unity.
 
-2. Choose **SpatialOS** > **Local launch** or press **Ctrl-L**
-
-3. In the Editor, open **Assets** > **Playground** > **Scenes** > **SampleScene**.
-
-## Full version
+Follow [this link](https://improbable.io/get-spatialos) to sign up for one for free.
 
 ### Setting up your machine
 
 1. Install prerequisites:
 
     #### Windows
 
-	- [Unity 2018.2.0f2](https://unity3d.com/get-unity/download/archive) with Linux Build Support and Mac Build Support components selected during the installation process.
-	- [Visual Studio 2017](https://www.visualstudio.com/downloads/)
-	    > Within Visual Studio Installer, on the Workloads tab, select **Game development with Unity** and **.NET Core cross-platform development**. In the summary on the right, deselect **Unity 2017.2 64-bit Editor** (the SpatialOS GDK for Unity requires Unity 2018.2.0f2). Make sure **Visual Studio Tools for Unity** is selected.
-    - [.NET Core SDK (x64)](https://www.microsoft.com/net/download/) (verified with `2.1.3xx` and `2.1.4xx`)
-	- SpatialOS, using the the [SpatialOS installer](https://console.improbable.io/installer/download/stable/latest/win)
-	<br>This installs:
+	1. [Unity 2018.2.0](https://unity3d.com/get-unity/download/archive) 
+        - Make sure to select the following components:
+            - **Linux Build Support**
+            - **Mac Build Support**
+	2. [Visual Studio 2017](https://www.visualstudio.com/downloads/) (Verified with version `15.8.4`) or [Rider](https://www.jetbrains.com/rider/) (Verified with version `2018.2.1`)
+	    - If you are using Visual Studio, within the Visual Studio Installer, on the Workloads tab:
+            - Select **Game development with Unity** 
+            - Select **.NET Core cross-platform development** 
+            - In the summary on the right: 
+                - Deselect **Unity 2017.2 64-bit Editor** (the SpatialOS GDK for Unity requires Unity 2018.2.0f2). 
+                - Make sure **Visual Studio Tools for Unity** is selected.
+        - If you are using Rider: 
+            - Install the [**Unity Support** plugin](https://github.com/JetBrains/resharper-unity) for a better experience.
+    3. [.NET Core SDK (x64)](https://www.microsoft.com/net/download/) (Verified with version `2.1.3xx` or `2.1.4xx`)
+	4. SpatialOS, using the the [SpatialOS installer](https://console.improbable.io/installer/download/stable/latest/win). This installs:
 		- the [`spatial` CLI](https://docs.improbable.io/reference/13.0/shared/spatial-cli-introduction)
```

Comment:
are we using latest or 13.0 as links to the SpatialOS documentation

Reply:
I think latest - will go through and change